packages/@o3r/third-party/src/bridge/ab-testing/ab-testing-bridge.spec.ts

```diff
@@ -0,0 +1,45 @@
+import {firstValueFrom} from 'rxjs';
+import {AbTestBridge, AbTestBridgeInterface} from './ab-testing-bridge';
+describe('A/B testing bridge', () => {
+  it('should add a new bridge to the window', (complete) => {
+    const bridgeName = 'ab-test';
+    const readyEventName = 'ab-test-ready';
+    expect((window as any)[bridgeName]).toBe(undefined);
+    document.addEventListener(readyEventName, () => {
+      complete();
+    }, {once: true});
```

Comment:
is that needed ? it seems you're not doing it for the 2 others tests

Reply:
I want to test once that we receive an event once the bridge is instantiated - I don't need to test it twice

Reply:
there might be a better way to check that the event has been dispatched (without having to wait for a timeout in case of failure)
maybe something like
```typescript
    const dispatchEventSpy = jest.spyOn(document, 'dispatchEvent');

    new AbTestBridge<string>((a?: string, b?: string) => a === b, {bridgeName, readyEventName});

    expect(dispatchEventSpy).toHaveBeenCalledWith(expect.any(Event));
    expect(dispatchEventSpy.mock.calls[0][0].type).toBe(readyEventName);
```